config/configuration.go

```diff
@@ -61,7 +64,7 @@ func Load() (*EnvConfig, []Config, error) {
 	cfg := getDefaultEnvConfig()
 	err := loader.Load(context.Background(), cfg)
 	if err != nil {
-		panic(err)
+		logrus.Fatalf("error returned when passing config into loader.Load(): %+v")
```

Comment:
Could you please fix the missing `err`?

```suggestion
		log.Fatalf("error returned when passing config into loader.Load(): %v", err)
```

Reply:
heh, not my best patch.